src/Paket.Core/Versioning/FrameworkHandling.fs

```diff
@@ -276,6 +276,46 @@ type SilverlightVersion =
         | SilverlightVersion.V5 -> "v5.0"
 
 [<RequireQualifiedAccess>]
+type MonoAndroidVersion =
+    | V1
+    | V22
+    | V23
+    | V403
+    | V41
+    | V43
+    | V44
+    | V5
+    | V6
+    | V7
+    | V71
+    member this.ShortString() =
+        match this with
+        | MonoAndroidVersion.V1    -> ""
+        | MonoAndroidVersion.V22   -> "2.2"
+        | MonoAndroidVersion.V23   -> "2.3"
+        | MonoAndroidVersion.V403  -> "4.03"
```

Comment:
Good catch, considering the [offical list of Xamarin.Android releases](https://developer.xamarin.com/releases/android/mono_for_android_4/mono_for_android_4.0.3/)  it's certainly 4.0.3 and my Xamarin Studio also gives this in the list of available target frameworks. Will fix that.

![image](https://user-images.githubusercontent.com/4660060/27288894-716b990c-5508-11e7-85ea-099ba0d66740.png)